src/main/java/Calculator.java

```diff
@@ -1,59 +1,59 @@
-public class Calculator {
+import java.util.Scanner;
 
+public class Calculator {
+    public static final int infinite = 999999;
 
-    public int sum(int a, int b) {
-        return a + b;
+    public int add(int i, int j) {
+        return i + j;
     }
 
-    public int subtract(int a, int b) {
-        return a - b;
+    public int subtract(int i, int j) {
+        return i - j;
     }
 
-    public int multiple(int a, int b) {
-        return a * b;
+    public int multiple(int i, int j) {
+        return i * j;
     }
 
-    public int divide(int a, int b) {
-        return a / b;
+    public int divide(int i, int j) {
+        return i / j;
     }
 
-    public char[] makeArrayFunction(String function) {
-        char[] arrayFunction = new char[function.length()];
-        for (int i = 0; i < function.length(); i++) {
-            arrayFunction[i] = (function.charAt(i));
-        }
-        return arrayFunction;
+    public String getValue() {
+        Scanner scanner = new Scanner(System.in);
+        String value = scanner.nextLine();
+        return value;
     }
 
-    public int changeNumber(char alphabet) {
-        return (int) (alphabet - '0');
+    public String[] splitString(String value){
+        String[] values = value.split(" ");
+        return values;
     }
 
-    public int matchFunction(char sign, int answer, int nextNumber) {
-        int nowAnswer = 0;
-        if (sign == '+') {
-            nowAnswer = sum(answer, nextNumber);
-        }
-        if (sign == '-') {
-            nowAnswer = subtract(answer, nextNumber);
-        }
-        if (sign == '*') {
-            nowAnswer = multiple(answer, nextNumber);
-        }
-        if (sign == '/') {
-            nowAnswer = divide(answer, nextNumber);
-        }
-        return nowAnswer;
+    public int calculate(int before, String operator, int after) {
+        int result = infinite;
```

Comment:
```suggestion
        int result = Integer.MAX_VALUE;
```
자바에서는 Integer 라는 클래스에 상수값으로 맥스 값이 구현되어있으니 가져다가 쓰시면 됩니다. infinite 라는 변수를 calculator라는 클래스가 필드로 가지고 있을 필요가 없게됩니다.